9/Dockerfile

```diff
@@ -63,11 +64,26 @@ RUN mkdir -p /var/run/redis && \
     greenbone-nvt-sync && \
     greenbone-scapdata-sync && \
     greenbone-certdata-sync && \
-    BUILD=true /start && \
+    BUILD=true /start && \ 
     service openvas-scanner stop && \
     service openvas-manager stop && \
     service openvas-gsa stop && \
     service redis-server stop
 
+RUN apt-get update && \
+    apt-get install -y postfix supervisor libsasl2-modules syslog-ng-core mailutils && \
```

Comment:
Why do you need supervisor, syslog and mailutils?

Also can you move these in with the first `apt-get install` around `14`, I see no reason to leave them separate.